awswrangler/cleanrooms/_read.py

```diff
@@ -82,13 +96,33 @@ def read_sql_query(
     >>>     output_bucket='output-bucket',
     >>>     output_prefix='output-prefix',
     >>> )
+
+    >>> import awswrangler as wr
+    >>> df = wr.cleanrooms.read_sql_query(
+    >>>     analysis_template_arn='arn:aws:cleanrooms:...',
+    >>>     params={'param1': 'value1'},
+    >>>     membership_id='membership-id',
+    >>>     output_bucket='output-bucket',
+    >>>     output_prefix='output-prefix',
+    >>> )
     """
     client_cleanrooms = _utils.client(service_name="cleanrooms", session=boto3_session)
 
+    if sql:
+        sql_parameters: ProtectedQuerySQLParametersTypeDef = {
```

Comment:
Should we put `ProtectedQuerySQLParametersTypeDef` in quotation marks? Since the import is conditional, and we don't require customers do install boto3 stubs to use `awswrangler`, I'm worried this line will fail for customers.

Reply:
Good catch - thanks